packages/components/addon/components/hds/copy/button/index.hbs

```diff
@@ -0,0 +1,16 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Button
+  class="{{this.classNames}}"
+  @size={{this.size}}
+  @isFullWidth={{@isFullWidth}}
+  @text={{@text}}
+  @icon={{this.icon}}
+  @isIconOnly={{@isIconOnly}}
+  @color="secondary"
+  @iconPosition="trailing"
+  {{clipboard text=this.textToCopy target=this.targetToCopy onError=this.onError onSuccess=this.onSuccess}}
```

Comment:
[note] we'll need to pull ember-cli-clipboard in to `dependencies` if we're relying on it here